go.mod

```diff
@@ -72,7 +70,6 @@ require (
 	github.com/lithammer/dedent v1.1.0
 	github.com/lpabon/godbc v0.1.1 // indirect
 	github.com/magiconair/properties v1.8.1 // indirect
-	github.com/mattn/go-isatty v0.0.9 // indirect
```

Comment:
dropping this is what required the drop to v0.0.4 in a later commit

Reply:
yes.